Services/AzineMailgunMailerService.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+
+namespace Azine\MailgunWebhooksBundle\Services;
+
+use Monolog\Logger;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class AzineMailgunMailerService
+{
+    private $mailer;
+    private $validator;
+    private $logger;
+    private $twig;
+    private $mailerUser;
+    private $ticketId;
+    private $ticketSubject;
+    private $ticketMessage;
+    private $adminUserEmail;
+    
+    public function __construct(
+        \Swift_Mailer $mailer,
+        ValidatorInterface $validator,
+        Logger $logger,
+        \Twig_Environment $twig,
+        $mailerUser,
+        $ticketId,
+        $ticketSubject,
+        $ticketMessage,
+        $adminUserEmail
+    ) {
+        $this->mailer = $mailer;
+        $this->validator = $validator;
+        $this->logger = $logger;
+        $this->twig = $twig;
+        $this->mailerUser = $mailerUser;
+        $this->ticketId = $ticketId;
+        $this->ticketSubject = $ticketSubject;
+        $this->ticketMessage = $ticketMessage;
+        $this->adminUserEmail = $adminUserEmail;
+    }
+
+    public function  sendSpamComplaintNotification($eventId)
+    {
+        if (is_null($this->validateEmail($this->adminUserEmail))) {            
```

Comment:
move this ($this->validateEmail($this->adminUserEmail)" before the "if" and assign it to a variable, so you don't have to call it twice.